elements/pfe-styles/src/pfe-typography-classes.scss

```diff
@@ -1,14 +1,15 @@
 @import "../../pfe-sass/pfe-sass";
 
 // TITLES
-@each $size in (6xl, 5xl, 4xl, 3xl, 2xl, xl, lg, md, sm, xs) {
+@each $size in ("6xl", "5xl", "4xl", "3xl", "2xl", xl, lg, md, sm, xs) {
   .pfe-title--#{$size}  {
-      @extend %pfe-title--#{$size}; 
+      @include pfe-typography($sizing: #{$size}, $important: true); 
   }
 }
+
 // TEXT
-@each $size in (lg, md, sm, xs) {
-  .pfe-content--text-#{$size}  {
-      @extend %pfe-content--text-#{$size}; 
+@each $size in (xl, lg, md, sm, xs) {
+  .pfe-text--#{$size}  {
+      @include pfe-typography(#{$size}, $type: text, $base: true, $important: true);  
```

Comment:
```suggestion
      @include pfe-typography($size, text, $base: true, $important: true);  
```

Reply:
You don't need to and I would recommend against interpolating inside a mixin call